src/shared/hooks/usePlugins.ts

```diff
@@ -46,7 +46,11 @@ export default function usePlugins() {
       fetch(`${pluginsPath as string}/manifest.json`, {
         signal: abortController.signal,
       })
-        .then(resp => resp.json())
+        .then(async resp => {
+          const data = await resp.json();
+          console.log('@@data', data);
```

Comment:
If we plan to leave these logs in the final version of this PR, then I'd suggest logging `@@Plugins Data` instead of `@@data`

Reply:
i thought i remove all logs, It's not meant to be their, i will remove it again

Reply:
:+1: